optaplanner-operator/src/test/java/org/optaplanner/operator/impl/solver/OptaPlannerSolverReconcilerTest.java

```diff
@@ -6,64 +6,73 @@
 
 import java.util.List;
 import java.util.Map;
+import java.util.UUID;
 
+import javax.enterprise.event.Observes;
 import javax.inject.Inject;
 
-import org.junit.jupiter.api.AfterEach;
+import org.assertj.core.api.SoftAssertions;
 import org.junit.jupiter.api.BeforeEach;
 import org.junit.jupiter.api.Test;
+import org.optaplanner.operator.impl.solver.model.AmqBroker;
 import org.optaplanner.operator.impl.solver.model.ConfigMapDependentResource;
 import org.optaplanner.operator.impl.solver.model.OptaPlannerSolver;
 import org.optaplanner.operator.impl.solver.model.OptaPlannerSolverSpec;
 import org.optaplanner.operator.impl.solver.model.Scaling;
-import org.optaplanner.operator.impl.solver.model.messaging.MessageAddress;
+import org.optaplanner.operator.impl.solver.model.keda.ScaledObject;
+import org.optaplanner.operator.impl.solver.model.keda.ScaledObjectDependentResource;
+import org.optaplanner.operator.impl.solver.model.keda.SecretTargetRef;
+import org.optaplanner.operator.impl.solver.model.keda.Trigger;
+import org.optaplanner.operator.impl.solver.model.keda.TriggerAuthentication;
+import org.optaplanner.operator.impl.solver.model.keda.TriggerAuthenticationDependentResource;
 
 import io.fabric8.kubernetes.api.model.ConfigMap;
+import io.fabric8.kubernetes.api.model.SecretKeySelector;
 import io.fabric8.kubernetes.api.model.apps.Deployment;
-import io.fabric8.kubernetes.client.server.mock.KubernetesServer;
+import io.fabric8.kubernetes.client.KubernetesClient;
 import io.javaoperatorsdk.operator.Operator;
+import io.quarkus.runtime.StartupEvent;
 import io.quarkus.test.junit.QuarkusTest;
-import io.quarkus.test.kubernetes.client.KubernetesTestServer;
-import io.quarkus.test.kubernetes.client.WithKubernetesTestServer;
 
-@WithKubernetesTestServer
 @QuarkusTest
-public class OptaPlannerSolverReconcilerTest {
-
-    @KubernetesTestServer
-    KubernetesServer mockServer;
+public class OptaPlannerSolverReconcilerTest extends AbstractKubernetesTest {
 
     @Inject
-    Operator operator;
+    private Operator operator;
 
-    @BeforeEach
-    void startOperator() {
+    // TODO: file an to quarkus-operator-sdk to enable operator.start() and .stop() in @BeforeEach and @AfterEach.
```

Comment:
File an what?
An how about doing it instead of littering the code with it?

Reply:
Good point